ingenialink/ethercat/servo.py

```diff
@@ -202,28 +202,25 @@ def _handle_sdo_exception(
              timeout period.
 
         """
-        if isinstance(
```

Comment:
Is Sdos the only place pysoem exceptions are raised? I would imagine that the process data could yield something similar...
In general I think this formatting would belong to pysoem, that we can contribute to. If not maybe put the formatting somewhere more global.
I.E. All WkErrors should be formatted in the same way, regardless where they happen.

Reply:
Only SDO operations can raise exceptions.